types/node/index.d.ts

```diff
@@ -6818,3 +6819,205 @@ declare module "http2" {
+		 * whose performanceEntry.name is equal to name, and optionally, whose performanceEntry.entryType is equal to type.
+		 * @param name
+		 * @param type
+		 * @return list of all PerformanceEntry objects
+		 */
+		getEntriesByName?(name: string, type?: string): PerformanceEntry[];
+
+		/**
+		 * Returns a list of all PerformanceEntry objects in chronological order with respect to performanceEntry.startTime
+		 * whose performanceEntry.entryType is equal to type.
+		 * @param type
+		 * @return list of all PerformanceEntry objects
+		 */
+		getEntriesByType?(type: string): PerformanceEntry[];
+
+		/**
+		 * Creates a new PerformanceMark entry in the Performance Timeline.
+		 * A PerformanceMark is a subclass of PerformanceEntry whose performanceEntry.entryType is always 'mark',
+		 * and whose performanceEntry.duration is always 0.
+		 * Performance marks are used to mark specific significant moments in the Performance Timeline.
+		 * @param name
+		 */
+		mark?(name?: string): void;
+
+		/**
+		 * Creates a new PerformanceMeasure entry in the Performance Timeline.
+		 * A PerformanceMeasure is a subclass of PerformanceEntry whose performanceEntry.entryType is always 'measure',
+		 * and whose performanceEntry.duration measures the number of milliseconds elapsed since startMark and endMark.
+		 *
+		 * The startMark argument may identify any existing PerformanceMark in the the Performance Timeline, or may identify
+		 * any of the timestamp properties provided by the PerformanceNodeTiming class. If the named startMark does not exist,
+		 * then startMark is set to timeOrigin by default.
+		 *
+		 * The endMark argument must identify any existing PerformanceMark in the the Performance Timeline or any of the timestamp
+		 * properties provided by the PerformanceNodeTiming class. If the named endMark does not exist, an error will be thrown.
+		 * @param name
+		 * @param startMark
+		 * @param endMark
+		 */
+		measure?(name: string, startMark: string, endMark: string): void;
+
+		/**
+		 * An instance of the PerformanceNodeTiming class that provides performance metrics for specific Node.js operational milestones.
+		 */
+		readonly nodeTiming?: PerformanceNodeTiming;
+
+		/**
+		 * Returns the current high resolution millisecond timestamp.
+		 * @return timestamp in ms
+		 */
+		now?(): number;
+
+		/**
+		 * The timeOrigin specifies the high resolution millisecond timestamp from which all performance metric durations are measured.
+		 */
+		readonly timeOrigin?: number;
+
+		/**
+		 * Wraps a function within a new function that measures the running time of the wrapped function.
+		 * A PerformanceObserver must be subscribed to the 'function' event type in order for the timing details to be accessed.
+		 * @param fn
+		 */
+		timerify<T extends (...optionalParams: any[]) => any>(fn: T) => T;
+	}
+
+	const performance: Performance;
+	const performanceEntry: PerformanceEntry;
+	const performanceNodeTiming: PerformanceNodeTiming;
```

Comment:
I don't think a `performanceNodeTiming` property is present on exports of `perf_hooks`.